nav2_smac_planner/src/a_star.cpp

```diff
@@ -63,14 +63,18 @@ void AStarAlgorithm<NodeT>::initialize(
   const int & terminal_checking_interval,
   const double & max_planning_time,
   const float & lookup_table_size,
-  const unsigned int & dim_3_size)
+  const unsigned int & dim_3_size,
+  const GoalHeadingMode & goal_heading_mode)
 {
   _traverse_unknown = allow_unknown;
   _max_iterations = max_iterations;
   _max_on_approach_iterations = max_on_approach_iterations;
   _terminal_checking_interval = terminal_checking_interval;
   _max_planning_time = max_planning_time;
-  NodeT::precomputeDistanceHeuristic(lookup_table_size, _motion_model, dim_3_size, _search_info);
+  _current_goal_heading_mode = goal_heading_mode;
+  NodeT::setGoalHeadingMode(_current_goal_heading_mode);
```

Comment:
For all, change `_current_goal_heading_mode`, `current_goal_heading_mode`, etc to just `goal_heading_mode`, if its set, its current :wink: No reason to add that

Reply:
bad habbits of mine :laughing:

Reply:
Honestly I wouldn't have said anything except for the fact that none of the other parameters or variables have "current" in them!